src/api/orders.js

```diff
@@ -0,0 +1,68 @@
+import BASEURL from "./index";
+
+export const getAllOrders = async () => {
+    try {
+        const response = await fetch(`${BASEURL}/order`, {
+            headers: {
+            "Content-Type": "application/json"
+      }});
+    
+        const result = await response.json();
+        return result;
+    } catch {
+        console.error(error);
+    }
+};
+
+export const createOrder = async (orderId) => {
+   try {
+    const response = await fetch(`${BASEURL}/order`, {
+        method: "POST",
+        headers: {
+        "Content-Type": "application/json"
+    },
+        body: JSON.stringify({
+        orderId: orderId,
+        cart: cart,
+    }),
+    }); 
+        const result = await response.json();
+        return result;
+    } catch (error) {
+        console.error(error)
+    }
+};
+
+export const updateOrder = async (id, status, cart) => {
+    try {
+        const response = await fetch(`${BASEURL}/order/${orderId}`, {
+            method: "PATCH",
+            headers: {
+                "Content-Type": "application/json"
+        },
+            body: JSON.stringify({
+                orderId: orderId,
```

Comment:
orderId key is fine, but the value orderId isn't an argument. The same goes for line 38 where orderId isn't defined. You'll have to change the argument "id to orderId"